feature_engine/imputation/__init__.py

```diff
@@ -18,4 +19,5 @@
     "AddMissingIndicator",
     "RandomSampleImputer",
     "DropMissingData",
+    "BaseImputer",
```

Comment:
```suggestion
```

Reply:
In the init we include the files that are for external use. BaseImputer is for internal use.